### PR TITLE
Hent klagebehandlinger

### PR DESCRIPTION
**Pull request by throndi**
https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-9405

### CODE REVIEW

File: src/frontend/App/hooks/useHentKlagebehandlinger.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export const useHentKlagebehandlinger = (): IProps => {
+    const { axiosRequest } = useApp();
+    const [klagebehandlinger, settKlagebehandlinger] = useState<Ressurs<Klagebehandlinger>>(
+        byggTomRessurs()
+    );
+    const { toggles } = useToggles();
+    const hentKlagebehandlinger = useCallback(
+        (fagsakPersonid: string) => {
+            if (toggles[ToggleName.visOpprettKlage]) {
```
Comment (charliemidtlyng) [inline src/frontend/App/hooks/useHentKlagebehandlinger.ts:21]: Usikker på om vi egentlig trenger toggles her. Om vi henter data som gir "tomt svar" så er vel det innafor?

File: src/frontend/App/typer/fagsak.ts [hunk 2]
```diff
@@ -59,31 +60,19 @@ export enum BehandlingResultat {
...
     DELVIS_TILBAKEBETALING: 'Delvis tilbakebetaling',
     FULL_TILBAKEBETALING: 'Full tilbakebetaling',
-    HENLAGT: 'Henlagt',
-};
-
-export const BehandlingResultatInkludertTilbakekreving = {
-    ...BehandlingResultat,
-    ...TilbakekrevingBehandlingsresultatstype,
+    MEDHOLD: 'Medhold',
+    IKKE_MEDHOLD: 'Ikke medhold',
```
Comment (charliemidtlyng) [inline src/frontend/App/typer/fagsak.ts:77]: `IKKE_MEDHOLD_FORMKRAV_AVVIST: 'Ikke medhold, formkrav avvist'`

File: src/frontend/App/typer/klage.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    resultat: KlagebehandlingResultat | undefined;
+    status: KlagebehandlingStatus;
+    vedtaksdato: string | undefined;
+    årsak: KlageÅrsak | undefined;
+}
+
+export enum KlagebehandlingResultat {
+    MEDHOLD = 'MEDHOLD',
+    IKKE_MEDHOLD = 'IKKE_MEDHOLD',
+    IKKE_SATT = 'IKKE_SATT',
```
Comment (charliemidtlyng) [inline src/frontend/App/typer/klage.ts:21]: `IKKE_MEDHOLD_FORMKRAV_AVVIST`

**Review by charliemidtlyng — APPROVED** [review]

File: src/frontend/App/typer/klage.ts (hunk 3 above)
Comment (ViktorGSolberg) [reply src/frontend/App/typer/klage.ts:22]: lagt til [b127242](https://github.com/navikt/familie-ef-sak-frontend/pull/1582/commits/b127242d3b93f7e79fa86780fba140f428bc9b3f)

File: src/frontend/App/typer/fagsak.ts (hunk 2 above)
Comment (ViktorGSolberg) [reply src/frontend/App/typer/fagsak.ts:77]: lagt til [b127242](https://github.com/navikt/familie-ef-sak-frontend/pull/1582/commits/b127242d3b93f7e79fa86780fba140f428bc9b3f)

File: src/frontend/App/hooks/useHentKlagebehandlinger.ts (hunk 1 above)
Comment (ViktorGSolberg) [reply src/frontend/App/hooks/useHentKlagebehandlinger.ts:21]: beholder denne inntil videre

**Pull request merged**